contract/views/contract_portal_templates.xml

```diff
@@ -227,56 +227,68 @@
-                                            </td>
-                                            <td name="td_price_unit" class="text-right">
-                                                <span
-                                                    t-field="line.price_unit"
-                                                    t-options='{"widget": "monetary", "display_currency": contract.currency_id}'
-                                                />
-                                            </td>
-                                            <td
-                                                name="td_recurring_interval"
-                                                class="text-right"
+                                        <tr
+                                            t-att-class="'bg-200 font-weight-bold o_line_section' if line.display_type == 'line_section' else 'font-italic o_line_note' if line.display_type == 'line_note' else ''"
+                                        >
+                                            <t t-if="not line.display_type">
+                                                <td name="td_name">
+                                                    <span t-field="line.name" />
+                                                </td>
+                                                <td
+                                                    name="td_quantity"
+                                                    class="text-right"
+                                                >
+                                                    <span t-field="line.quantity" />
+                                                </td>
+                                                <td
+                                                    name="td_price_unit"
+                                                    class="text-right"
+                                                >
+                                                    <span
+                                                        t-field="line.price_unit"
+                                                        t-options='{"widget": "monetary", "display_currency": contract.currency_id}'
+                                                    />
+                                                </td>
+                                                <td
+                                                    name="td_recurring_interval"
+                                                    class="text-right"
+                                                >
+                                                    <span
+                                                        t-field="line.recurring_interval"
+                                                    />
+                                                    <t
+                                                        t-esc="dict(line._fields['recurring_rule_type'].selection).get(line.recurring_rule_type)"
```

Comment:
Have you check if this becomes translated according language?

Reply:
I have changed and work fine now in multi-languages.